.github/workflows/release.yaml

```diff
@@ -1,375 +0,0 @@
-name: Release
-on:
-  workflow_dispatch:
-    inputs:
-      version:
-        type: string
-        description: The version you intend to release without leading 'v' (eg x.y.z and not vx.y.z)
-  pull_request:
-    types: [closed]
-  push:
-    branches:
-      - "release/**"
-      - "next/**"
-    tags:
-      - v*
-  release:
-    types: [created]
-
-env:
-  VERSION: ${{ github.event.inputs.version }}
-  APP_ID: 251311
```

Comment:
We should perhaps store this ID in a secret to be safer? It's been here for probably a year by now, but it can be done in a follow up.